source/kernel/Acceptor.cpp

```diff
@@ -52,12 +60,17 @@ void Acceptor::handle_event(int fd, uint32_t events)
             throw std::runtime_error("accept() failed: " +
                                      std::string(strerror(errno)));
         ConnectionHandler* conn_handler =
-            new ConnectionHandler(conn_fd, _reactor);
-        _reactor->register_handler(conn_fd, conn_handler, EPOLLIN | EPOLLOUT);
+            new ConnectionHandler(conn_fd, _reactor, _config, _server_id);
+        _reactor->register_handler(conn_fd, conn_handler, EPOLLIN | EPOLLET);
```

Comment:
The EPOLLOUT is for output event but the EPOLLET is for edge trigger enevts

https://github.com/LeaYeh/webserver/blob/main/doc/non-blocking%20IO.md#epoll

Reply:
tbh I'm not sure can I remove EPOLLIN or not